### ISSUE TITLE
Only one report (doc) generated instead of 3

### ISSUE THREAD

**Issue by scream314** [glenjamin/mocha-multi#54]
Running my tests with
```
mochaOptions: ['--reporter-options', 'spec=-,mocha-bamboo-reporter=./test/mocha-bamboo-reporter.json,doc=./test/docs.html,mocha-junit-reporter=./test/mocha-junit-reporter.xml'],
```
I get the `spec ` output, the `docs.html` file gets generated with the correct report, but the files `mocha-junit-reporter.xml` and `mocha-bamboo-reporter.json` are empty.

Tried to run with only `mochaOptions: ['--reporter-options', 'spec=-,mocha-bamboo-reporter=./test/mocha-bamboo-reporter.json'],` with the same results.

Is there a way to have all 4 outputs?

**Comment by glenjamin** [comment]
Do the other 2 reporters print onto standard out? If not then you won’t be able to use mocha multi to control their output destination

**Comment by scream314** [comment]
@glenjamin 
I am not sure, but probably not.
I also tried to have 3 outputs using `mocha-multi-reporters`, but it was not working either (I think it was only able to generate the `junit` report).

Is there any way to have at least `spec`, `mocha-bamboo-reporter` and `mocha-junit-reporter` at the same time without running the tests 3 times?

**Comment by glenjamin** [comment]
I think those reporters take a `reporterOptions` param themselves, which declares their output file.

I've just merged a PR and released `v1.1.0` which allows you to specify reporter options via a JSON file which might give you a way to make this work.
